schema/adminschema/v1.json

```diff
@@ -44,7 +44,26 @@
                   "type": "boolean"
               }
           }
-      }
+      },
+      "ClientCreateRequest": {
+          "id": "ClientCreateRequest",
+          "type": "object",
+	      "description": "'client' field is a client registration request as defined by the OpenID Connect dynamic registration spec, and holds fields such as redirect URLs, prefered algorithms, etc. For brevity field names and types of that object have been omitted.",
+          "properties": {
```

Comment:
Seems to work okay.

https://github.com/ericchiang/dex/blob/admin_client_registration/schema/adminschema/README.md#clientcreaterequest